_inc/client/my-plan/checklist-progress-card/progress-bar/style.scss

```diff
@@ -0,0 +1,52 @@
+.progress-bar {
```

Comment:
This might be a little too generic class name to have at global scope.

Reply:
Good point. I rolled all of this into a specific component and specific styles now.